fontforge/featurefile.c

```diff
@@ -5128,6 +5130,26 @@ return;
 	}
     }
 
+    /* A ligature substitution may also have a single destination. In which case */
+    /*  it will look just like a single substitution. So if there are both */
+    /*  ligature and single subs in a lookup, translate all the singles into */
+    /*  ligatures */
+    has_single = has_ligature = false;
```

Comment:
Can we just add ligature searching as a third prong in the previous block (alongside the single/multiple check)?

Reply:
And maybe we could add error reporting for multiple-to-multiple substitutions? You've reviewed the code in question more recently than I have; tell me if this is unnecessary.

Reply:
Completely separate from the issue of how these various changes may eventually get packaged together, for sanity's sake I'm going to continue this discussion in push request #3350 , where I can update the code specific to this question.